adapter-jdbc/src/main/java/io/nosqlbench/adapter/jdbc/optypes/JDBCExecuteOp.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2023 nosqlbench
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.nosqlbench.adapter.jdbc.optypes;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class JDBCExecuteOp extends JDBCOp {
+    private final static Logger logger = LogManager.getLogger(JDBCExecuteOp.class);
+
+    public JDBCExecuteOp(Connection connection, Statement statement, String queryString) {
+        super(connection, statement, queryString);
+    }
+
+    @Override
+    public void run() {
+        logger.debug(() -> "Executing JDBCExecuteOp for the given cycle.");
+        try {
+            this.getConnection().setAutoCommit(false);
+            if (logger.isDebugEnabled()) {
+                logger.debug(() -> "JDBC Query is: " + this.getQueryString());
+            }
+            boolean isResultSet = this.getStatement().execute(this.getQueryString());
+            if (isResultSet) {
+                logger.debug(() -> ">>>>>>>>>>Executed a SELECT operation [" + this.getQueryString() + "]<<<<<<<<<<");
+            } else if (!isResultSet) {
```

Comment:
This is slightly confusing, and is the fall-through case anyway. We can simplify it.
In general, there is lots of branching based on diagnostic features. We might want to remove or reduce these. It's generally better to compute the values which will be used, and then allow the logging subsystem to decide whether to show (compute) from the closure.

Reply:
I've implemented an update version. Please re-review. TY